### PR TITLE
[FIX] Markdown message preview

### PR DESCRIPTION
**Pull request by LucasSiqz**
@RocketChat/ReactNative

Before changes:

image

image

Image preview wrong and preview markdown messages with two white spaces

After changes: 

image

image

### CODE REVIEW

File: app/containers/markdown/index.js [hunk 1]
```diff
@@ -151,7 +151,7 @@ export default class Markdown extends PureComponent {
 				{literal}
 			</Text>
 		);
-	}
+	};
```
Comment (diegolmello) [inline app/containers/markdown/index.js]: Can you remove these semicolons?

File: app/containers/markdown/index.js [hunk 2]
```diff
@@ -317,7 +324,7 @@ export default class Markdown extends PureComponent {
 
 		// Ex: '[ ](https://open.rocket.chat/group/test?msg=abcdef)  Test'
 		// Return: 'Test'
-		m = m.replace(/^\[([\s]]*)\]\(([^)]*)\)\s/, '').trim();
+		m = m.replace(/(\s)\[([\s]]*)\]\(([^)]*)\)\s/, '').trim();
```
Comment (diegolmello) [inline app/containers/markdown/index.js]: What are you doing here?

File: app/presentation/RoomItem/LastMessage.js [hunk 3]
```diff
@@ -35,7 +35,7 @@ const formatMsg = ({
 		prefix = `${ lastMessage.u.username }: `;
 	}
 
-	let msg = `${ prefix }${ lastMessage.msg.replace(/[\n\t\r]/igm, '') }`;
+	let msg = `${ prefix }${ lastMessage.msg.replace(/[\n\t\r]/igm, '').trim() }`;
```
Comment (diegolmello) [inline app/presentation/RoomItem/LastMessage.js]: What are you doing here?

File: app/containers/markdown/index.js (hunk 1 above)
Comment (LucasSiqz) [reply app/containers/markdown/index.js]: yes, it was my prettier extension, sorry xD

File: app/presentation/RoomItem/LastMessage.js (hunk 3 above)
Comment (LucasSiqz) [reply app/presentation/RoomItem/LastMessage.js]: it was just a unsuccessfully tried to remove the two white spaces in markdown message preview that i forgot to remove.

File: app/containers/markdown/index.js (hunk 2 above)
Comment (LucasSiqz) [reply app/containers/markdown/index.js]: I saw that \s covers spaces and added it to regex and apparently worked for this case, but seeing better now, it is failing in the example above. Do you have any suggestion on how to solve this problem?

**Comment by diegolmello** [comment]
@LucasSiqz I removed your images from description because of inappropriate language. Feel free to add another. Thanks.

**Comment by LucasSiqz** [comment]
> @LucasSiqz I removed your images from description because of inappropriate language. Feel free to add another. Thanks.

I updated the images, sorry :/

**Pull request closed without merging**